Device/Program.cs

```diff
@@ -10,12 +7,18 @@ namespace MessageSample
 {
     public class Program
     {
-        public static async Task<int> Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            var configuration = Configuration.BuildConfiguration();
-            var device = new Device(configuration);
-            await device.RunSampleAsync(ApplicationMessageReceived);
-            return 0;
+            var configuration = Configuration.BuildConfiguration();
+            var iotHubDeviceConnectionString = configuration["IotHubDeviceConnectionString"];
+            var device = new Device(iotHubDeviceConnectionString);
+
+            await device.ConnectDevice();
+            await device.SubscribeToEventAsync(ApplicationMessageReceived);
+
+            Console.WriteLine("Enter message payload for D2C");

```

Comment:
Could we at least put this in a while loop with a command to exit? I'd like to keep the functionality of continuously sending messages